reflect/build.gradle.kts

```diff
@@ -24,28 +24,29 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.internal.gradle.publish
 
-/**
- * A DSL element of [SpinePublishing] extension which allows disabling publishing
- * of [protoJar] artifact.
- *
- * This artifact contains all the `.proto` definitions from `sourceSets.main.proto`. By default,
- * it is published.
- *
- * Take a look on [SpinePublishing.protoJar] for a usage example.
- *
- * @see [registerArtifacts]
- */
-class ProtoJar {
+import io.spine.internal.dependency.Kotlin
+import io.spine.internal.gradle.checkstyle.CheckStyleConfig
+import io.spine.internal.gradle.javadoc.JavadocConfig
+import io.spine.internal.gradle.report.license.LicenseReporter
 
-    /**
-     * Set of modules, for which a proto JAR will not be published.
-     */
-    var exclusions: Set<String> = emptySet()
+plugins {
+    module
+   `maven-publish`
+    `kotlin-jvm-module`
+    `dokka-for-kotlin`
+    idea
+    `project-report`
+    `detekt-code-analysis`
```

Comment:
Can `kotlin-jvm-module` apply `module` on its own? It looks like `kotlin-jvm-module` is already `module` but with some additional stuff.

And some of these plugins are already applied in `module`. This block can be shortened.